### PR TITLE
Do more to prevent fee-rate-disagreement force-closures

### PR DESCRIPTION
**Pull request by luckysori**
We recently saw another one in production after increasing the background confirmation target from 12 to 24.

### CODE REVIEW

File: crates/ln-dlc-node/src/fee_rate_estimator.rs [hunk 1]
```diff
@@ -9,6 +9,14 @@ use std::sync::RwLock;
 use std::sync::RwLockReadGuard;
 use std::sync::RwLockWriteGuard;
 
+const CONFIRMATION_TARGETS: [(ConfirmationTarget, usize); 3] = [
+    // We choose an extremely high background confirmation target to avoid force-closing channels
+    // unnecessarily.
+    (ConfirmationTarget::Background, 1000),
```
Comment (holzeis) [inline crates/ln-dlc-node/src/fee_rate_estimator.rs:15]: @luckysori why not `1008`? 😅

**Review by holzeis — APPROVED** [review]
Let's get this in as quick as possible

File: crates/ln-dlc-node/src/fee_rate_estimator.rs (hunk 1 above)
Comment (luckysori) [reply crates/ln-dlc-node/src/fee_rate_estimator.rs:15]: I didn't get that it was meant to be 7 days!

**Pull request merged**